### DISCUSSION TITLE
Unable to get AGH and Unbound working in Docker

### DISCUSSION THREAD

**Discussion by agneevX** [AdguardTeam/AdGuardHome#3297] [Q&A]
### Environment

* **Version of AdGuard Home server:** 0.106.3
* **How did you install AdGuard Home:** Docker

### Issue details

I have AdGuard Home and Unbound inside the same Docker network.

Unbound is set to listen to `0.0.0.0:5053`.

Adding `unbound:5053` to Upstream DNS servers does not seem to work.

**Comment by ameshkov** [comment]
Probably a duplicate of a different question.

AGH won't use DHCP hostnames, you need to specify the exact IP address. If it's the same Docker container, just use `127.0.0.1`
